CompareRaftMembers/DBListStream.js

```diff
@@ -49,6 +49,11 @@ class DBListStream extends stream.Transform {
             }
             [bucket, objectKey] = [key.slice(0, slashIndex), key.slice(slashIndex + 1)];
         }
+        // ignore special buckets, that cannot be listed from bucketd
+        // like regular buckets
+        if (bucket === 'users..bucket' || bucket.startsWith('mpuShadowBucket')) {
```

Comment:
want to confirm there's not an extra '.' in `users..bucket`

Reply:
Yes the name contains two dots